### PR TITLE
Simple code fixed in in pkg/kubelet

### PR DESCRIPTION
**Pull request by denkensk**
**What this PR does / why we need it**:
Simple code fixed in in pkg/kubelet

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
none

**Special notes for your reviewer**:
none

**Release note**:
```release-note
none
```

### CODE REVIEW

**Comment by yue9944882** [comment]
[https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#10-trivial-edits](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#10-trivial-edits)

thanks for contributing! plz don't split trival pulls. 🙂  u can take an issue tagged `good first issue` if u r trying contributing out of curiosity. PRs are always welcome! thx!

/hold

**Comment by denkensk** [comment]
@yue9944882  I fix #68026 in kubelet

File: pkg/kubelet/reason_cache.go [hunk 1]
```diff
@@ -40,8 +40,8 @@ type ReasonCache struct {
 	cache *lru.Cache
 }
 
-// Reason is the cached item in ReasonCache
-type reasonItem struct {
+// ReasonItem is the cached item in ReasonCache
+type ReasonItem struct {
```
Comment (vishh) [inline pkg/kubelet/reason_cache.go]: Does this have to be public?

Comment (juanvallejo) [reply pkg/kubelet/reason_cache.go]: Agree, leave this private unless there's an explicit need to make it public

Comment (juanvallejo) [reply pkg/kubelet/reason_cache.go]: Otherwise, lgtm

Comment (denkensk) [reply pkg/kubelet/reason_cache.go]: ok, I change it to private.

**Review by feiskyer — COMMENTED** [review]
LGTM. But it seems no typo fixes as clarified in the title?

**Comment by denkensk** [comment]
@feiskyer  I change the title.

**Comment by denkensk** [comment]
@feiskyer  can you help me to merge the pr ？thx

**Comment by feiskyer** [comment]
/lgtm

Please note that code freeze has started now. The merge queue would only accept critical PRs for v1.12 until Sep. 19.

**Comment by Huang-Wei** [comment]
@denkensk It'd be good to squash the 3 commits into one - you can run `git rebase -i HEAD~3` to squash into one commit, then `git push origin <your branch> -f`

**Pull request merged**